app/src/main/java/com/hammer/talkbbokki/presentation/detail/DetailScreen.kt

```diff
@@ -180,22 +148,16 @@ fun DetailFlipCard(
             coroutineScope {
                 launch {
                     animate(
-                        initialValue = 180f,
-                        targetValue = 0f,
-                        animationSpec = tween(
-                            durationMillis = 540,
-                            easing = FastOutSlowInEasing
+                        initialValue = 180f, targetValue = 0f, animationSpec = tween(
+                            durationMillis = 540, easing = FastOutSlowInEasing
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)